README.md

```diff
@@ -1,3 +1,28 @@
 # thursday
 
 Classifying radio galaxies as Fanaroff-Riley Type I (FR-I) and Fanaroff-Riley Type II (FR-II).
+
+
+## Input files
+
+You will need
+- The data used in https://arxiv.org/pdf/1705.03413.pdf. This is stored in `asu.tsv`.
```

Comment:
Are FRICAT/FRIICAT acquired automatically by the ipynb file?

Reply:
You are the one who wrote it...

Reply:
I don't know